sapi/fpm/fpm/fpm_unix.h

```diff
@@ -5,9 +5,9 @@
 
 #include "fpm_worker_pool.h"
 
-int fpm_unix_resolve_socket_premissions(struct fpm_worker_pool_s *wp);
-int fpm_unix_set_socket_premissions(struct fpm_worker_pool_s *wp, const char *path);
-int fpm_unix_free_socket_premissions(struct fpm_worker_pool_s *wp);
+int fpm_unix_resolve_socket_permissions(struct fpm_worker_pool_s *wp);
+int fpm_unix_set_socket_permissions(struct fpm_worker_pool_s *wp, const char *path);
+int fpm_unix_free_socket_permissions(struct fpm_worker_pool_s *wp);
```

Comment:
These are probably api breaks and would need to be dropped

Reply:
I don't think those are API breaks and this goes to the master only anyway so it should be fine.